app/src/main/java/ch/epfl/sdp/appart/MainActivity.java

```diff
@@ -38,7 +38,9 @@ protected void onCreate(Bundle savedInstanceState) {
         startActivity(intent);
     }
 
-    //public static boolean isDemoMode() {
-    //    return DEMO_MODE;
-    //}
+    /*
+    public static boolean isDemoMode() {
+        return DEMO_MODE;
+    }
```

Comment:
I think we can rid of this. I'll take care of it in my PR so you don't have to push again.

Reply:
Yeah thanks! you never know on Cirrus what could happen 🙄